docs/UPGRADING.rst

```diff
@@ -0,0 +1,20 @@
+Upgrading HaaS
+==============
+
+This file describes the procedure for upgrading HaaS to a new version.
+
+1. Read the release notes, as they may contain version-specific information.
+2. Stop the HaaS services (webserver and HaaS network daemon).
+3. Download and install the new version of HaaS::
+
+     $ git clone https://github.com/cci-moc/haas
+     $ python setup.py install
+
+3. Upgrade the database::
+
+     $ haas-admin db upgrade heads
```

Comment:
The alembic docs go into the full detail (please do read them), but basically it says to upgrade *all* branches (one per extension with scripts, plus haas core) to the latest version. The default is head, which will spit out an error if there's more than one.

I don't know that we want to go into the full detail in this document. `haas-admin db` has a lot of stuff that's mostly of interest to developers. It might make sense to move that to a `haas-dev` command and write a few custom sub-commands to `haas-admin` doing just the basics.